src/SharpAudio.Codec/Decoder.cs

```diff
@@ -17,6 +17,12 @@ public abstract class Decoder : IDisposable
         /// Specifies the length of the decoded data. If not available returns 0
         /// </summary>
         public virtual TimeSpan Duration => TimeSpan.FromSeconds((float) _numSamples / (_audioFormat.SampleRate * _audioFormat.Channels));
+        
+        
+        /// <summary>
+        /// Specifies the current position of the decoded data. If not available returns 0
```

Comment:
Not sure if returning 0 as a default is a good idea, since that could be a valid value. Maybe add a bool property `HasPosition`? Otherwise we could leave it like that